### DISCUSSION TITLE
pause HFSS simulation

### DISCUSSION THREAD

**Discussion by z7766918** [ansys/pyaedt#4081] [Q&A] [answered]
Hi everyone,
I created a HFSS work process by using pyaedt, but I've hit a bump in the road, because I want to pause the HFSS simulation, save project, and close AEDT, and then restart simulation after thirty minutes; but I didn't get any information about "pause" in document, how should I do to realize "pause" function?

Ansys version : 2022R2
Pyaedt version : 0.6.75
Python version : 3.9.15
OS : Windows 10

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @z7766918,

AEDT API in 2022R2 does not allow to pause of the simulation.

In 2023R2, you can run the simulation and not block the script (blocking parameter).

https://aedt.docs.pyansys.com/version/0.7/API/_autosummary/pyaedt.hfss.Hfss.analyze_setup.html#pyaedt.hfss.Hfss.analyze_setup

If you need to pause the simulation, please ask Ansys technical support, and they will submit an enhancement request to the Ansys dev. team.
